build/repo.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <ProjectsToPack Include="$(RepositoryRoot)src\OrchardCore\*\*.csproj" />
```

Comment:
Where/how is this file used.
How is it better than a filter in the appveryor file?